src/pint/models/fdjump.py

```diff
@@ -0,0 +1,207 @@
+"""System and frequency dependent delays to model profile evolution."""
+
+import re
+from warnings import warn
+
+import astropy.units as u
+import numpy as np
+
+from pint.models.parameter import boolParameter, maskParameter
+from pint.models.timing_model import DelayComponent
+
+fdjump_max_index = 20
+
+
+class FDJump(DelayComponent):
+    """A timing model for system-dependent frequency evolution of pulsar
+    profiles.
+
+    This model expresses the delay as a polynomial function of the
+    observing frequency/logarithm of observing frequency in the SSB frame.
+    This is intended to compensate for the delays introduced by frequency-dependent
+    profile structure when a different profiles are used for different systems.
+
+    The default behavior is to have FDJUMPs as polynomials of the observing
+    frequency (rather than log-frequency). This is different from the convention
+    used for global FD parameters. This choice is made to be compatible with tempo2.
+    This is controlled using the `FDJUMPLOG` parameter. `FDJUMPLOG Y` may not be
+    tempo2-compatible.
+
+    Caveat
```

Comment:
Would this be better as "Note" which I think gets formatted differently by RTD (e.g., https://nanograv-pint--1615.org.readthedocs.build/en/1615/_autosummary/pint.toa_select.TOASelect.html#pint.toa_select.TOASelect)?  

(I think in many cases we have "Notes", but it might be that "Note" is preferred and those should be updated in another PR)

Reply:
Changed this to a note box.